### ISSUE TITLE
SSL/TLS

### ISSUE THREAD

**Issue by gitmatt19** [k9mail/k-9#4992]
Servers are starting to block K9 because it does not support a high enough TLS level.

In particular unless you downgrade now you can longer access Dovecot servers because it requires at least TLS 1.2 by default.

**Comment by cketti** [comment]
K-9 Mail uses Android's TLS stack. Modern Android versions support TLS 1.2 just fine.

**Comment by cketti** [comment]
The next stable K-9 Mail version will only run on Android 5.0 an newer. These Android versions have TLS 1.2 enabled by default.
See https://developer.android.com/reference/kotlin/javax/net/ssl/SSLSocket#protocols

**Comment by gitmatt19** [comment]
After seeing this I was going to post all the logs showing that at least for me K9 Android 4.4 & 5 will only work at SSLv3 (I have them showing connections rejected for anything above it) but I decided to start poking around Dovecot (CentOS8). I found no changes in the configuration other than having to drop the TLS level to SSLv3. I decided to rebuild all of the certificates for no good reason but did it anyway. I verified everything still worked , re-accepting the new cert on K9 etc. I then bumped the TLS level up to 1.1 and restarted DoveCot and checked email again and after about a minute of spinning it connected and got email, after that the connections were as fast as normal. No idea why how/why they K9/Dovecot got stuck at SSLv3 but it seems to work good now,
